src/_resources/fact-sheet-minimum-account-requirements-for-transit-riders.md

```diff
@@ -0,0 +1,112 @@
+---
+date: "2023-08-01T17:00:00-07:00"
+title: "Fact Sheet: Minimum Account Requirements for Transit Riders"
```

Comment:
@mrose914 since this is filed under `Case Studies` -- shouldn't the title reflect that? It's a little confusing to see this listed with the other case studies but using the Fact Sheet title? (this was taken directly from the ticket #311 but I'm just noticing on the preview site it looks a little strange)

Reply:
cc @allejo